core/server/worker/src/main/java/alluxio/worker/dora/PagedDoraWorker.java

```diff
@@ -270,9 +287,10 @@ public FileInfo getFileInfo(String ufsFullPath, GetStatusPOptions options) throw
     DoraMeta.FileStatus status = mUfsStatusCache.getIfPresent(ufsFullPath);
     if (syncIntervalMs >= 0 && status != null) {
       // Check if the metadata is still valid.
-      if (System.currentTimeMillis() - status.getTs() > syncIntervalMs) {
+      if (System.nanoTime() - status.getTs()  > syncIntervalMs * Constants.MS_NANO) {
```

Comment:
extra space?
```suggestion
      if (System.nanoTime() - status.getTs() > syncIntervalMs * Constants.MS_NANO) {
```

Reply:
fixed in https://github.com/Alluxio/alluxio/pull/17194